.yarnrc.yml

```
@@ -0,0 +1,13 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: '@yarnpkg/plugin-workspace-tools'
+  - path: .yarn/plugins/@yarnpkg/plugin-compat.cjs
+    spec: '@yarnpkg/plugin-compat'
+
+yarnPath: .yarn/releases/yarn-berry.cjs
+
+# use "global cache" so when switching between branches, packages will not be removed from cache so they will not have to be redownloaded
+globalFolder: './.yarn/global'
+enableGlobalCache: true
```

```suggestion
enableGlobalCache: true
```

You can have `enableGlobalCache: true` without the `globalFolder` config so it uses the default one and shares it between all projects on your machine

Default paths to the cache:
```
Linux   - ~/.yarn/berry/cache
Windows - ~/AppData/Local/Yarn/Berry/cache
```